Visibility.md

```diff
@@ -4,17 +4,23 @@
 
 This section will describe wgsl enhancements to control which WGSL elements are visible to importers.
 
-* which wgsl elements are available to import from wgsl modules within the same package?
-* which wgsl elements are available to import from wgsl modules in other packages?
 * how to re-export elements so that they're visible with a different path or name?
-* lib.wgsl to make things visible at the root of a package?
 * controlling host visible names like entry points and overrides?
 * Should export allow `as` renaming?
-* Treat .wgsl files as .wesl with every element exported?
 * Why not export struct Foo?
-  * Many current wgsl parsers (including wgpu's naga) would 
-    choke on the unknown attribute as is and feels like having 
+  * Many current wgsl parsers (including wgpu's naga) would
+    choke on the unknown attribute as is and feels like having
     two export forms is a bit inconsistent.
+* Consider changing to public by default for imports within the package only.
```

Comment:
I like this idea, especially with the prospect of importing wgsl/glsl etc. files locally and exposing them later at the package level :)

Reply:
As I was drafting, I thought that the arguments for default private vs default public within the package were fairly even on both sides. I'm sure we'll discuss it further.